packages/mason_logger/lib/src/link_wrap.dart

```diff
@@ -0,0 +1,8 @@
+/// Helper function for "wrapping" an optional message with link.
+String linkWrap(String? message, Uri uri) {
```

Comment:
```suggestion
String link({required Uri uri, String? message}) {
```

wdyt?

Reply:
Looks good! I think that makes it more clean how you can use it